daemon/stats.go

```diff
@@ -0,0 +1,29 @@
+package daemon
+
+import (
+	"encoding/json"
+
+	"github.com/docker/docker/api/stats"
+	"github.com/docker/docker/engine"
+)
+
+func (daemon *Daemon) ContainerStats(job *engine.Job) engine.Status {
+	s, err := daemon.SubscribeToContainerStats(job.Args[0])
```

Comment:
Unless I'm missing it, this might be lacking some logic to support short container IDs.